Notes/Codes.md

```diff
@@ -2,6 +2,8 @@
 
 ## Sparse vs Dense
 
+If a code makes use of less than or equal to half of its available codewords its said to be sparse, otherwise the code is  dense. Binary codes have $2^N$ available codewords for $N$ bits so, in this case, codes that have $ \le 2^{N - 1}$ codewords would be deemed sparse.
```

Comment:
You need to remove the space at the beggining of the equation, as it won't display properly on github. I would also exlplicitly say $\frac{2^{n}}{2}$